src/workers/customCodes.ts

```diff
@@ -186,4 +187,14 @@ Suggestion: ${balancedTime}`,
   UncaughtArgumentType: (): ErrorCode => {
     return { id: CustomErrorCodes.Type.UNCAUGHT_ARG, message: `Error: Command Argument Type Mismatch` };
   },
+  /**
+   * uncheckable warning code and message.
+   */
+  UncheckableArgumentType: (): ErrorCode => {
+    return {
+      id: CustomErrorCodes.Type.UNCHECK_ARG,
+      message: `Warning: Not able to evaluate identifiers.
```

Comment:
@goetzrrGit Does it make sense to say something like "Warning: Not able to evaluate identifiers in order to provide validation"?

Reply:
How about "not able to evaluate and validate identifier"?

Reply:
perfect!

Reply:
@duranb It should be good to go now